artifacts/definitions.py

```diff
@@ -22,24 +25,32 @@
     'Containerd': 'Containerd artifacts',
     'Docker': 'Docker artifacts.',
     'Execution': 'Contain execution events.',
-    'ExternalAccount': (
+    'ExternalAccount': '<Deprecated>', # TODO: to be deleted
+    'External Account': (
         'Information about any user accounts e.g. username, '
         'account ID, etc.'),
     'External Media': 'Contain external media data or events e.g. USB drives.',
+    'File System': 'Artifacts related to the file system e.g. MFT.',
     'Hadoop': 'Hadoop artifacts.',
     'IM': 'Instant Messaging / Chat applications artifacts.',
+    'Interactive': 'Artifacts created by interactive (e.g. RDP) user activity.',
```

Comment:
This label might be misleading, since a lot of "interactive" artifacts can be script as well.

Reply:
This label refers to artifacts that would be generated through GUI interaction with the system (e.g. ShellBags in Windows), which to you point could be both automated and manual. What do you suggest -- make the comment more clear, rename the label to something different?

Reply:
How do you want to use this label?